app/src/main/java/org/gdg/frisbee/android/arrow/Organizer.java

```diff
@@ -0,0 +1,42 @@
+package org.gdg.frisbee.android.arrow;
+
+import com.google.android.gms.plus.model.people.Person;
+
+public class Organizer {
```

Comment:
We have lots of POJOs like this. It can be easily confused with others I think. Even if it is in the arrow package. What do you think?